snap/snapcraft.yaml

```diff
@@ -0,0 +1,29 @@
+name: prqlc # you probably want to 'snapcraft register <name>'
+title: PRQL Compiler
+base: core20 # the base snap is the execution environment for this snap
+version: git
+summary: PRQL is a modern language for transforming data
+description: |
+  PRQL is a modern language for transforming data — a simple, powerful,
+  pipelined SQL replacement.
+  It lets you transpile PRQL queries into SQL queries.
```

Comment:
```suggestion
  PRQL is a modern language for transforming data — a simple, powerful,
  pipelined SQL replacement.
```

I think we should stick to the standard description across all the releases. If we think the standard description isn't sufficiently descriptive, we can discuss at that level

Reply:
I think we should include some some more paragraphs to better explain what it is and what it does and what it can do. A "one line" thing is good, but these descripion fields allow multiple paragraphs that we can use to help convey information better than just a oneliner can do alone.

I also think it should be mentioned that it is a CLI app and that the binary is `prqlc`else you might install this application and you might not know how to run it because it does not show up in any menu.